src/components/Workshops/Workshop.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import '../../styles/Workshops.css';
+import ReactPlayer from 'react-player/youtube';
+import { Github, Youtube, File } from '@geist-ui/icons';
+
+
+export default function Workshop({ title, youtube, author, description, readme, slides}) {
+
+    
+    const README =  readme ?
+                    <button className='workshop-button'>
+                        <Github color='var(--hackAccent)' size={28} />
+                        <a className='workshop-link' href={readme} target='_blank' rel='noreferrer'>&nbsp; Readme</a>
+                    </button> : null;
+
+    const SLIDES =  slides ? 
+                    <button className='workshop-button'>
+                        <File color='var(--hackAccent)' size={28} />
+                        <a className='workshop-link' href={slides} target='_blank' rel='noreferrer'>&nbsp; Slides</a>
+                    </button> : null;

```

Comment:
This is a really simple way of rendering conditional components that I've not thought of doing before, but totally makes sense now that I think about it. Nice job! I need to steal this next time for some projects of my own.